barretenberg/cpp/src/barretenberg/honk/proof_system/eccvm_prover.hpp

```diff
@@ -63,12 +60,10 @@ template <ECCVMFlavor Flavor> class ECCVMProver_ {
 
     Polynomial quotient_W;
 
-    work_queue<Curve> queue;
-
     sumcheck::SumcheckOutput<Flavor> sumcheck_output;
     pcs::gemini::ProverOutput<Curve> gemini_output;
     pcs::shplonk::ProverOutput<Curve> shplonk_output;
-    std::shared_ptr<PCSCommitmentKey> pcs_commitment_key;
+    std::shared_ptr<PCSCommitmentKey> commitment_key;
```

Comment:
Couldn't help m,yself here. polynomial commitment scheme commitment key is a ridiculous name. I could also live with `pcs_key`